project-base/app/config/packages/twig.yaml

```diff
@@ -2,6 +2,8 @@ twig:
     debug: "%kernel.debug%"
     exception_controller: null
     form_themes:
+        - '@ShopsysFramework/Admin/Form/Product/Price/PricesWithCalculatedSellingPricesType.html.twig'
+        - '@ShopsysFramework/Admin/Form/Product/Price/ProductPricesWithVatSelectType.html.twig'
```

Comment:
usually, we name the templates without the first capital letter, just saying :smile:

Reply:
renamed :) 
the usual way of naming in symfony application is snake_case, maybe we should discuss it in the future